db/dbformat.h

```diff
@@ -31,6 +31,8 @@ static const size_t kL0_CompactionTrigger = 6;
 // Level-0 (any overlapped level) number of files where a grooming
 //     compaction could start
 static const size_t kL0_GroomingTrigger = 4;
+static const size_t kL0_GroomingTrigger10min = 2;
```

Comment:
I do not disagree.  But "static const" is part of the original Google writing.  I see no benefit to removing.  Is there a benefit beyond removing redundancy of declaration?